src/components/Settings/settings.less

```diff
@@ -0,0 +1,239 @@
+@import "../../less/vars.less";
+
+@expandable-arrow-size: 15px;
+@expandable-margin: 40px;
+
+
+.settingsPage {
+  // &__markets {
+  //   background-color: @bg-color-muted;
+  //   padding: 40px 0;
+  // }
+
+  // &__stats {
+  //   margin: 50px 0;
+  // }
+
+  &__header {
+    margin: @page-heading-margins;
+  }
+}
+
+.settingsGeneral, .settingsModerators {
+
+  &__icon {
+    width: 35px;
+    height: 35px;
+    position: relative;
+    margin-top: 7px;
+  }
+
+  &::after {
+    content: '';
+    display: table;
+    clear: both;
+  }
+}
+
+.moderatorItem {
+  margin-top: 20px;
+}
+
+
+.settingsSection {
+  margin-bottom: 50px;
+}
+
+.settingsHeading {
+  margin-bottom: 20px;
+
+  &__icon {
+    display: inline-block;
+    height: 50px;
+    width: 50px;
+    background: @btn-gradient;
+    vertical-align: middle;
+    text-align: center;
+    color: @btn-font-color;
+    line-height: 50px;
+    font-size: 24px;
+    border-radius: 50%;
+  }
+
+  &__title {
+    display: inline-block;
+    padding-left: 20px;
+    line-height: 50px;
+    vertical-align: middle;
+  }
+}
+
+// .marketInfos {
+//   text-align: right;
+//   position: relative;
+// }
+
+// .marketInfo__info {
+//   overflow: hidden;
+//   text-overflow: ellipsis;
+//   padding-left: 100px; //just until we use real names
+//   margin-bottom: 20px;
+// }
+
+// .marketInfo__info--value {
+//   font-size: 15px;
+//   color: @font-color-dark;
+//   letter-spacing: 1px;
+//   font-weight: 400;
+//   margin: 0;
+//   margin-bottom: 2px;
+// }
+
+// .marketInfo__info--label {
+//   text-transform: uppercase;
+//   color: @font-color-dark;
+//   font-size: 10px;
+//   opacity: .6;
+//   letter-spacing: .5px;
+// }
+
+// .marketTitle__heading {
+//   margin: @page-heading-margins;
+// }
+
+// .marketDescription {
+//   margin-bottom: 40px;
+// }
+
+// .marketDescription__text {
+//   font-size: 18px;
+//   line-height: 30px;
+//   letter-spacing: .25px;
+//   font-weight: 300;
+// }
+
+// .marketTimer {
+//   margin-top: 12px;
+//   margin-bottom: 12px;
+// }
+
+// .marketTimer__live {
+//   font-size: 28px;
+//   font-weight: 300;
+//   letter-spacing: 1px;
+//   line-height: 1;
+// }
+
+// .marketTime__absolute {
+//   color: @font-color-dark;
+//   opacity: .6;
+//   font-size: 10px;
+//   text-transform: uppercase;
+//   letter-spacing: .5px;
+// }
+
+// .marketControls.container {
+//   padding-top: 40px;
+// }
+
+// .marketControls__button {
+//   margin-right: 20px;
+// }
```

Comment:
Can you remove these comments here? If they're not needed anymore, just remove them. Commenting out code can be avoided because we use git :)